.github/workflows/build-with-bal-test-native.yml

```diff
@@ -36,10 +36,10 @@ jobs:
       - name: Set up GraalVM
         uses: graalvm/setup-graalvm@v1
         with:
-          version: 'latest'
           java-version: '17.0.7'
```

Comment:
@TharmiganK Is it okay to use the version `17.0.7` instead of `17`?

Reply:
Using the version 17 will always pull the latest one using this action. Currently the latest one is `17.0.8`. I think `17.0.7` is set since lang team has verified the changes with that version. I prefer to set `17` here, so that we can make sure that the GraalVM builds are working with the latest version.

Reply:
+1. @gabilang FYA.

Reply:
Ack.